showcase/showcase-index.js

```diff
@@ -0,0 +1,35 @@
+import AxesShowcase from './showcase-sections/axes-showcase';
```

Comment:
Dunno if we can use the `export from` syntax with our current setup? It might require us to `import * as obj` if we want to grab the full `showcasesWrapper` that I think you're doing somewhere